tests/components/local_calendar/test_diagnostics.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for diagnostics platform of local calendar."""
+
+from freezegun import freeze_time
+
+from homeassistant.core import HomeAssistant
+
+from .conftest import TEST_ENTITY, ClientFixture
+
+from tests.common import MockConfigEntry
+from tests.components.diagnostics import get_diagnostics_for_config_entry
+from tests.typing import ClientSessionGenerator
+
+TEST_ICS = """BEGIN:VCALENDAR
+PRODID:-//github.com/allenporter/ical//4.5.0//EN
```

Comment:
Note: I'm considering sending a follow up PR to set a PRODID specific to home-assistant, rather than the underlying library. (Also, there was previously a bug here where PROID was never set fixed in ical 4.5.0. Also i'd not like this